demos/src/pa11y.mustache

```diff
@@ -0,0 +1,319 @@
+<!DOCTYPE html>
+<html lang="en" class="o-hoverable-on ">
+<head>
+	<meta charset="utf-8" />
+	<meta http-equiv="X-UA-Compatible" content="IE=Edge" />
+	<title>o-footer: footer pa11y demo</title>
+	<meta name="viewport" content="initial-scale=1.0, width=device-width" />
+	<script src="//polyfill.webservices.ft.com/v1/polyfill.min.js?features=html5,svg,datauri,default"></script>
+	<style>body { margin: 0; } .core .o--if-js, .enhanced .o--if-no-js { display: none; }</style>
+	<script>(function(d) { d.className = d.className + ' demo-js'; })(document.documentElement);</script>
+	<link rel="stylesheet" href="demo.css" />
+</head>
+<body>
+<footer class="o-footer o-footer--theme-light" data-o-component="o-footer" data-o-footer--no-js>
+	<div class="o-footer__container">
+		<div class="o-footer__copyright" role="contentinfo">
+			<small>
+				Markets data delayed by at least 15 minutes. &#169; THE FINANCIAL TIMES LTD .
+				<abbr title="Financial Times" aria-label="F T">FT</abbr> and ‘Financial Times’ are trademarks of The Financial Times Ltd.<br>
+				The Financial Times and its journalism are subject to a self-regulation regime under the <a href="http://www.ft.com/editorialcode" aria-label="F T Editorial Code of Practice">FT Editorial Code of Practice</a> .
+			</small>
+		</div>
+	</div>
+	<div class="o-footer__brand">
+		<div class="o-footer__container">
+			<div class="o-footer__brand-logo"></div>
+		</div>
+	</div>
+</footer>
+
+<footer class="o-footer o-footer--theme-light-hover" data-o-component="o-footer" data-o-footer--no-js>
```

Comment:
Commented in the CSS below but if this is just a style for the pa11y test it shouldn't use the `o-footer` style classnames